paddle/fluid/inference/api/analysis_predictor.cc

```diff
@@ -427,12 +444,27 @@ std::unique_ptr<PaddlePredictor> CreatePaddlePredictor<
   }
 
   std::unique_ptr<PaddlePredictor> predictor(new AnalysisPredictor(config));
-  if (!dynamic_cast<AnalysisPredictor *>(predictor.get())->Init(nullptr)) {
+  auto predictor_p = dynamic_cast<AnalysisPredictor *>(predictor.get());
+
+  if (!predictor_p->Init(nullptr)) {
     return nullptr;
   }
+
+  if (config.quantizer_enabled()) {
+    if (!predictor_p->Quantize()) {
+      return nullptr;
+    }
+  }
```

Comment:
how about line453-456 
```
if (config.quantizer_enabled() && !predictor_p->Quantize()) {
   return nullptr;
}
```

Reply:
fixed in https://github.com/PaddlePaddle/Paddle/pull/16396